typescript/cli/src/tests/commands/helpers.ts

```diff
@@ -230,12 +232,96 @@ export async function extendWarpConfig(params: {
     warpDeployPath,
   );
   warpDeployConfig[chainToExtend] = extendedConfig;
+  // Remove remoteRouters and destinationGas as they are written in readWarpConfig
```

Comment:
Like mentioned in another comment, I would like to avoid reading the config if we can. If that has to happen in another PR, so be it